serialization-avro/README.md

```diff
@@ -0,0 +1,57 @@
+# Serialization with Apache Avro
+
+This sample demonstrates usage of Apache Avro for messaging and storage of events inside AxonServer.
+For commands, events and queries and query responses the Avro schema definitions are provided and are located 
+in `src/main/avro` directory. 
+
+Here is an example for `CardIssuedEvent` event:
+
+```avroschema
+{
+    "type": "record",
+    "namespace": "io.axoniq.dev.samples.serializationavro.api",
+    "name": "CardIssuedEvent",
+    "doc": "A new gift card is issued.",
+    "revision": "1",
+    "javaAnnotation": "org.axonframework.serialization.Revision(\"1\")",
+    "fields": [
+        {
+            "name": "id",
+            "type": "string"
+        },
+        {
+            "name": "amount",
+            "type": "int"
+        }
+    ]
+}
+```
+
+Using the `avro-maven-plugin`, the Java classes are generated into `target/generated-sources/avro` directory 
+during the `generate-sources` build phase. 
+
+The remaining part of the application is a standard implementation of a gift card application, that uses the 
+`AvroSerializer` by applying configuration of the serializer using application properties.
+
+```yaml
+
+axon:
+  serializer:
+    events: avro
+    messages: avro
+    general: jackson

```

Comment:
Bit of a nit, but I can imagine people might end up here without reading the docs. In doing so, they might think "why not set the `general` serializer to Avro too?" 

So, I was thinking, perhaps it's worthwhile to add a one liner comment here, stating Avro is not supported as the general serializer. What do you think?

Reply:
They can try, it will throw a nicely formulated exception :-)

Reply:
Ha yes, that's true of course.